diff --git a/utils/build-script-impl b/utils/build-script-impl
index c003508..2716ceb 100755
--- a/utils/build-script-impl
+++ b/utils/build-script-impl
@@ -83,7 +83,7 @@ KNOWN_SETTINGS=(
     cmake                       ""               "path to the cmake binary"
     distcc                      ""               "use distcc in pump mode"
     distcc-pump                 ""               "the path to distcc pump executable. This argument is required if distcc is set."
-    build-runtime-with-host-compiler   ""        "use the host c++ compiler to build everything"
+    build-runtime-with-host-compiler   "1"        "use the host c++ compiler to build everything"
     cmake-generator             "Unix Makefiles" "kind of build system to generate; see output of 'cmake --help' for choices"
     verbose-build               ""               "print the commands executed during the build"
     install-prefix              ""               "installation prefix"
